### PR TITLE
Program certificate for different program types

### PR DESCRIPTION
**Pull request by annagav**
### What are the relevant tickets?
Fix https://github.com/mitodl/hq/issues/3370

### Description (What does it do?)
Uses program type to specify the type of program for which a certificate is issued.

Something to think about: omitting the Series might be a better solution.

### Screenshots (if appropriate):
Screenshot 2024-02-07 at 5 05 04 PM
Screenshot 2024-02-07 at 5 06 07 PM

### CODE REVIEW

**Review by JenniWhitman — APPROVED** [review]
Works as described.

**Comment by annagav** [comment]
@pdpinch does the wording work? 
`has successfully completed all courses and received passing grades to earn a Series program certificate in`

**Comment by pdpinch** [comment]
I asked the MITx core team and they said this is good as is. It's not ideal, but it's not worth holding up users program certificates and it's not worth the additional complexity.

**Review by pdpinch — APPROVED** [review]

**Pull request merged**
